martin/src/pg/pool.rs

```diff
@@ -30,8 +30,10 @@ const RECOMMENDED_POSTGRES_VERSION: Version = Version::new(12, 0, 0);
 pub struct PgPool {
     id: String,
     pool: Pool,
-    // When true, we can use margin parameter in ST_TileEnvelope
-    margin: bool,
+    /// Indicates if the margin parameter in `ST_TileEnvelope` is supported.
+    ///
+    /// This being `False` indicates that tiles may be cut off at the edges.
+    st_envelope_margin_is_supported: bool,
```

Comment:
perhaps `use_st_envelope_margin`?  A bit shorter and avoids a passive.

Reply:
maybe even remove the `st`? not certain

Reply:
That is a bit cryptic. 🤔

I your version from code below is better:
`supports_tile_margin`

Reply:
yep, i like this name too